pkg/controller/pod_volume_restore_controller.go

```diff
@@ -292,18 +347,16 @@ func (c *podVolumeRestoreController) processRestore(req *arkv1api.PodVolumeResto
 		}
 	}
 
-	cmd := exec.Command("/bin/sh", "-c", strings.Join([]string{"/complete-restore.sh", string(req.Spec.Pod.UID), volumeDir, string(restoreUID)}, " "))
-
-	if stdout, stderr, err = runCommand(cmd); err != nil {
-		log.WithError(errors.WithStack(err)).Errorf("Error running command=%s, stdout=%s, stderr=%s", resticCmd.String(), stdout, stderr)
-		return c.fail(req, fmt.Sprintf("error running restic restore, stderr=%s: %s", stderr, err.Error()), log)
+	// Create the .ark directory within the volume dir so we can write a done file
+	// for this restore.
+	if err := os.MkdirAll(filepath.Join(volumePath, ".ark"), 0755); err != nil {
```

Comment:
Would it be possible to have the done file go somewhere other than inside the actual volume?

Reply:
it would need to be somewhere that the init container could see it - could add an emptyDir volume to the pod specifically for done files

Reply:
Oh right this is the communications "protocol" between the DS pod and the actual pod.

What happens if you do
1. backup
1. restore (which creates the .ark done file)
1. backup
1. restore

Does the 2nd backup include the done file, and does it trick the 2nd restore into thinking it's finished before it actually is?

Reply:
It doesn't trick the restore - there's a .ark/ dir, but the file is named with the restore's UID, and the init container is given this UID and looks specifically for file with that name. I can add code to not copy over prior done files when doing a restore (had it before but it got dropped somewhere along the way)

Reply:
Ok cool, glad it's UID based. If you want to add the exclusion, that's fine, or you can just put in a TODO for now.

Reply:
added a TODO for now